about.html

```diff
@@ -7,57 +7,121 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
-
```

Comment:
Good job with this page.  Also try to get the nav to be the same size as the rest of the content on the page.